x-pack/plugins/security_solution/public/cloud_security_posture/components/misconfiguration/misconfiguration_overview.tsx

```diff
@@ -0,0 +1,175 @@
+import { buildMisconfigurationPreviewQuery } from '@kbn/cloud-security-posture-common';
+
+const getFindingsStats = (passedFindingsStats: number, failedFindingsStats: number) => {
+  if (passedFindingsStats === 0 && failedFindingsStats === 0) return [];
+  return [
+    {
+      key: i18n.translate(
+        'xpack.securitySolution.flyout.right.insights.misconfigurations.passedFindingsText',
+        {
+          defaultMessage: 'Passed findings',
+        }
+      ),
+      count: passedFindingsStats,
+      color: euiThemeVars.euiColorSuccess,
+    },
+    {
+      key: i18n.translate(
+        'xpack.securitySolution.flyout.right.insights.misconfigurations.failedFindingsText',
+        {
+          defaultMessage: 'Failed findings',
+        }
+      ),
+      count: failedFindingsStats,
+      color: euiThemeVars.euiColorVis9,
+    },
+  ];
+};
+
+const MisconfigurationEmptyState = ({ euiTheme }: { euiTheme: EuiThemeComputed<{}> }) => {
+  return (
+    <EuiFlexItem>
+      <EuiFlexGroup direction="column" gutterSize="none">
+        <EuiFlexItem>
+          <EuiTitle
+            size="m"
+            css={css`
+              font-weight: ${euiTheme.font.weight.bold};
+            `}
+          >
+            <h1>{'-'}</h1>
+          </EuiTitle>
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiText
+            size="m"
+            css={css`
+              font-weight: ${euiTheme.font.weight.semiBold};
+            `}
+            data-test-subj="noFindingsDataTestSubj"
+          >
+            <FormattedMessage
+              id="xpack.securitySolution.flyout.right.insights.misconfigurations.noFindingsDescription"
+              defaultMessage="No Findings"
+            />
+          </EuiText>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    </EuiFlexItem>
+  );
+};
+
+const MisconfigurationPreviewScore = ({
+  passedFindings,
+  failedFindings,
+  euiTheme,
+}: {
+  passedFindings: number;
+  failedFindings: number;
+  euiTheme: EuiThemeComputed<{}>;
+}) => {
+  return (
+    <EuiFlexItem>
+      <EuiFlexGroup direction="column" gutterSize="none">
+        <EuiFlexItem>
+          <EuiTitle
+            size="s"
+            css={css`
+              font-weight: ${euiTheme.font.weight.bold};
+            `}
+          >
+            <h1>{`${Math.round((passedFindings / (passedFindings + failedFindings)) * 100)}%`}</h1>
+          </EuiTitle>
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiText
+            size="m"
+            css={css`
+              font-weight: ${euiTheme.font.weight.semiBold};
+            `}
+          >
+            <FormattedMessage
+              id="xpack.securitySolution.flyout.right.insights.misconfigurations.postureScoreDescription"
+              defaultMessage="Posture score"
+            />
+          </EuiText>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    </EuiFlexItem>
+  );
+};
+
+export const MisconfigurationsOverview = ({ hostName }: { hostName: string }) => {
```

Comment:
Why `overview` and not `preview`?

Reply:
hmmm good question
can't quite remember why lol 
I think I just